src/brasil/gov/vcge/utils.py

```diff
@@ -1,4 +1,6 @@
 # -*- coding:utf-8 -*-
+# Flake8: noqa
```

Comment:
acho que isso aqui deve ser removido; qual o objetivo?

Reply:
Na linha 3 eu tentei ignorar somente a questão da ordem de importação # noqa: I001 mas o code-analysis não respeita isso conforme o manual do flake8. http://flake8.pycqa.org/en/stable/user/violations.html?highlight=In-line

E coloquei fora da ordem pelo texto

> For conversions from Python 3 code to Py2/3, use the pasteurize script instead. This converts Py3-only constructs (e.g. new metaclass syntax) to Py2/3 compatible constructs and adds __future__ and future imports to the top of each module.
http://python-future.org/overview.html#automatic-conversion-to-py2-3-compatible-code

Reply:
acabei de consertar.